students/KyleCreek/lesson03/assignment/src/basic_operations.py

```diff
@@ -0,0 +1,126 @@
+"""
+# Title: basic_operations.py
+# Desc: Establishes the basic operations for
+# Assignment03
+# Change Log: (Who, When, What)
+# KCreek, 4/20/2019, Created Script
+"""
+
+#from assignment.src.customer_model import *
+from assignment.src.customer_model import *
+import peewee
+import logging
+
+
+def search_customer(customer_id):
+    """
+    Returns a dictionary of Customer w/ Desired
+    Customer ID
+    :param customer_id: Customers ID
+    :return: Dictionary of  Customer
+    """
+    try:
+        # Query Customer Database for row
+        data = Customer.get(Customer.customer_id == customer_id)
+        # Place Data into a return dictionary
+        return_dict = {"First Name": data.first_name, "Last Name": data.last_name,
+                       "Email": data.email_address,"Phone Number": data.phone_number}
+        logging.info("The Following Dictionary was created: {}".format(return_dict))
+        return return_dict
+    # Exception where query target does not exist
+    except Exception as e:
```

Comment:
If we cannot find a record, is it really an exception? Maybe simple the record is not there...